src/state/tables.ts

```diff
@@ -61,6 +61,7 @@ export interface TableActions {
     tableName: string,
     schema: TableSchema,
     aidColumns: string[],
+    isOverwriting: boolean,
```

Comment:
Adds bloat in a few places. Always deleting would be fine since it queries the collections and does nothing if it doesn't exist.

Reply:
I had some reservations, but OK, I think it is better this way